andela/haven-ah-backend
# #161990616 Create seeder files for the models

#### What does this PR do?
Create seed data and seeder files for models.

#### Description of Task to be completed
- add dummy data files.
- add seeder files for models.

#### How should this be manually tested?
- on the terminal run `git clone https://github.com/andela/haven-ah-backend.git`.
- run `git checkout ch-seed-models-161990616`.
- ensure that PostgreSQL is running on your local machine.
- set up databases (test and development) following the `.env.sample` file.
- run `npm install` to install the npm packages.
To seed the database
- run `npm run db-migrate`  to migrate the tables.
- run `npm run db-seed` to seed the database.

#### Screenshots where applicable
<img width="1242" alt="screen shot 2018-11-19 at 1 01 48 am" src="https://user-images.githubusercontent.com/29674347/48679991-ceaf0b00-eb96-11e8-975b-e44b3dd0a65b.png">
<img width="572" alt="screen shot 2018-11-19 at 1 03 40 am" src="https://user-images.githubusercontent.com/29674347/48680009-06b64e00-eb97-11e8-83fc-f560e15368f2.png">

#### What are the relevant pivotal tracker stories?
#161990616
